src/objects.rs

```diff
@@ -16,7 +17,8 @@ pub type OneBasedIndex = std::num::NonZeroUsize;
 /// https://wiki.factorio.com/Blueprint_string_format#Blueprint_book_object
 pub struct BlueprintBook {
     pub item: String,
-    pub label: String,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub label: Option<String>,
```

Comment:
Whether or not this change makes sense depends on the origin of the test case you included. Did you generate that within the game? If yes, then the wiki documentation is out of date. If not, then we should add appropriate labels to the test instead of changing this field.

Reply:
Yes, that blueprint was generated using in-game with Factorio 1.0. It appears that (at least 3 weeks ago) the wiki was out of date.

Reply:
I'm willing to take your word on it.